File: Latihan B/C++/TimSepakBola.cpp
class TimSepakBola{

	//atribut private
	private:

		string nama_tim;
		string negara;
		string tahun_berdiri;
		string pemain[50];
		string pelatih;

	public:

		//konstruktor
		TimSepakBola(){

		}

		//getter dan setter nama tim
		void setNamaTim(string nama_tim){
			this->nama_tim = nama_tim;
		}

		string getNamaTim(){
			return this->nama_tim;
		}

		//getter dan setter asal negara tim
		void setNegara(string negara){
			this->negara = negara;
		}

		string getNegara(){
			return this->negara;
		}

		//getter dan setter tahun berdiri tim
		void setTahunBerdiri(string tahun_berdiri){
			this->tahun_berdiri = tahun_berdiri;
		}

		string getTahunBerdiri(){
			return this->tahun_berdiri;
		}

		//getter dan setter pemain
		void setPemain(int indeks, string pemain){
			this->pemain[indeks] = pemain;
		}

		string getPemain(int indeks){
			return this->pemain[indeks];
		}

		//getter dan setter pelatih
		void setPelatih(string pelatih){
			this->pelatih = pelatih;
		}

		string getPelatih(){
			return this->pelatih;
		}

		//destruktor
		~TimSepakBola(){

		}

};

File: Latihan B/PHP/TimSepakBola.php
<?php

class TimSepakBola{

	//atribut private
	private $nama_tim;
	private $negara;
	private $tahun_berdiri;
	private $pemain = array();
	private $pelatih;

	//konstruktor
	public function __construct(){

	}

	//getter dan setter untuk nama tim
	public function setNamaTim($nama_tim){
		$this->nama_tim = $nama_tim;
	}

	public function getNamaTim(){
		return $this->nama_tim;
	}

	//getter dan setter untuk asal negara tim
	public function setNegara($negara){
		$this->negara = $negara;
	}

	public function getNegara(){
		return $this->negara;
	}

	//getter dan setter untuk tahun berdiri tim
	public function setTahunBerdiri($tahun_berdiri){
		$this->tahun_berdiri = $tahun_berdiri;
	}

	public function getTahunBerdiri(){
		return $this->tahun_berdiri;
	}

	//getter dan setter untuk pemain
	public function setPemain($indeks, $pemain){
		$this->pemain[$indeks] = $pemain;
	}

	public function getPemain($indeks){
		return $this->pemain[$indeks];
	}

	//getter dan setter untuk pelatih
	public function setPelatih($pelatih){
		$this->pelatih = $pelatih;
	}

	public function getPelatih(){
		return $this->pelatih;
	}

	//destruktor
	public function __destruct(){

	}

}

?>